apps/script/checkEntryPoints.js

```diff
@@ -58,6 +58,7 @@ const SILENCED = [
   'flappy',
   'gamelab',
   'gamelab-api',
+  'header',
```

Comment:
please don't add this -- you should be able to avoid this by calling the new file `layouts/_header.js`. see this recent example: https://github.com/code-dot-org/code-dot-org/commit/9d4a96faaa8f76d48437348071a2659fd5a8aa48

Reply:
oh, neat! Thanks for the pointer